components/statsTracking/fn_addStatsEventHandlersToClass.sqf

```diff
@@ -0,0 +1,43 @@
+// Adds event handlers to the given unit, to track various stats for viewing on the debriefing.
+// These event handlers are local to the client, and the data they collect is relevant only 
+// to the client, so we don't need to sync the data across the network.
+
+// TODO Ensure these do run locally and not globally
+
+if (hasInterface) then {
+	["ace_firedPlayer", {
+		params ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile"];
+		systemChat "Player fired";
+		_currentShots = missionNamespace getVariable ["playerShots", createHashMap];
+		_currentCount = _currentShots getOrDefault [_ammo, 0];
+		_updated = _currentCount + 1;
+		_currentShots set [_ammo, _updated];
+		missionNamespace setVariable ["playerShots", _currentShots, false];
+	}] call CBA_fnc_addEventHandler;
+
+	["ace_unconscious", {
+		params ["_unit", "_state"];
+
+		if (_unit == player) then {
+			_currentUncons = missionNamespace getVariable ["playerUncons", 0];
+			_updated = _currentUncons + 1;
+			missionNamespace setVariable ["playerUncons", _updated, false];
```

Comment:
Perhaps rename the variable to something like `cafe_...` to make it clear it's a framework thing?